### ISSUE TITLE
Backup and restore commands needed?

### ISSUE THREAD

**Issue by dmorrison** [fnichol/knife-server#52]
I'm just getting started with knife-server, and I'm having trouble understanding why the backup and restore commands are needed. Can't you use `knife download` and `knife upload`?

Also, forgive my ignorance if I'm missing something obvious.

Thanks for the handy tool!

**Comment by fnichol** [comment]
No, you're not at all :smile:  These knife plugins simply pre-dated the knife-essentials being merged into Chef proper (and pre-dated my awareness of knife-essentials, I'd bet).

I'll cross-post my response just now on #31:

---

> Looking back at the backup/restore code, I'm fairly certain that Chef's knife download / and knife upload / may be better replacements that are supported out of the box. I'll make at least one more release with the backup/restore plugins intact, then may drop them in a major version bump. Sound reasonable?

**Comment by fnichol** [comment]
p.s. just marking this a duplicate so I can close them out when addressed. Thanks!

**Comment by dmorrison** [comment]
Makes sense - thanks!
